lib/homieDevice.js

```diff
@@ -13,13 +13,15 @@ const DEFAULT_CONFIG = {
   "mqtt": {
     "host": "localhost",
     "port": 1883,
-    "base_topic": "devices/",
+    "base_topic": "homie/",
```

Comment:
Is this part of the spec? If so, great. If not, I'd like to understand the thought behind this change.

Reply:
It is not required to be `homie`, but it is the [default base topic in the spec](https://homieiot.github.io/specification/#base-topic).  My thought here was that using the spec's default would better align with the defaults on homie controllers and can avoid manual configuration.

Reply:
Also, per the spec:

> Be aware, that only the default base topic homie/ is eligible for automatic discovery by third party controllers.

Reply:
Thanks - didn't know that about discoverability.